code/client/src/pages/Restore/RestoreByScan.jsx

```
@@ -89,7 +89,7 @@ const RestoreByScan = ({ isActive, onComplete, onCancel }) => {
     f()
   }, [addressInput])
 
-  const onScan = async (e, isJson) => {
+  const onScan = async (e, { isJson }) => {
```

This is causing restore by scanning seed QR code to silently fail